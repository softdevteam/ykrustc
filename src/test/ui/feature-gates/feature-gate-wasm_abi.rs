extern "wasm" fn foo() {
    //~^ ERROR: wasm ABI is experimental and subject to change
}

fn main() {
    foo();
}
