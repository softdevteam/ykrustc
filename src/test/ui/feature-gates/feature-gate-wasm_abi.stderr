error[E0658]: wasm ABI is experimental and subject to change
  --> $DIR/feature-gate-wasm_abi.rs:1:8
   |
LL | extern "wasm" fn foo() {
   |        ^^^^^^
   |
   = note: see issue #83788 <https://github.com/rust-lang/rust/issues/83788> for more information
   = help: add `#![feature(wasm_abi)]` to the crate attributes to enable

error: aborting due to previous error

For more information about this error, try `rustc --explain E0658`.
